nautobot/extras/forms.py

```diff
@@ -1390,10 +1384,15 @@ class StatusModelCSVFormMixin(CSVModelForm):
 
 class TagForm(NautobotModelForm):
     slug = SlugField()
+    content_types = MultipleContentTypeField(
+        label="Content Type(s)",
+        queryset=ModelSubclassesQuery("nautobot.core.models.generics.PrimaryModel").as_queryset,
+        required=False,
```

Comment:
Does it make sense to allow the user to create a Tag with no associated content-types? Or should this be a required field in the UI?